openapi_python_client/templates/property_templates/model_property.pyi

```diff
@@ -1,6 +1,12 @@
 {% macro construct(property, source, initial_value=None) %}
 {% if property.required and not property.nullable %}
+{% if source == "response.yaml" %}
+yaml = YAML(typ="base")
```

Comment:
We were previously using the `safe` loaded from PyYaml. Does that not work here?

```suggestion
yaml = YAML(typ="safe")
```